### DISCUSSION TITLE
Closing inactive issues?

### DISCUSSION THREAD

**Discussion by lionelfleury** [JanusGraph/janusgraph#2988] [General]
Issues count is more than 400. Many are from 2017.
Should some be closed for inactivity or no more relevant?
Spring clean-up somehow needed ;) 
Thanks guys!
Cheers,
Lionel

**Comment by vtslab** [comment]
Yes, closing issues that do not refer to the 0.5 or 0.6 branches is a good idea. It is important though that the closing message uses careful wording with appreciation for the issuer, the reason why it is closed and a note that the issue is still searchable on github.

**Comment by FlorianHockmann** [comment]
I think the question is whether we really want to close all old issues just because they haven't seen any activity in a while. If we want to do that, then we can probably just use a bot that automates this for us.
Old issues can however still be relevant. Just because nobody addressed them yet, doesn't mean that they aren't important and won't be addressed in the future. Closing them signals that we don't think they are relevant in my opinion.

Also what advantage do you see in closing old issues? If you just care for new issues, can't you simply only look at issues at the first few pages?

If we however want to close old issues only if they are no longer relevant or maybe already implemented, then we of course have to inspect all old issues manually which requires quite some work. @chupman did that once back in 2019: https://groups.google.com/g/janusgraph-dev/c/wDrBaxLcmRg/m/aQuzlCkrCAAJ

But I'm not completely against closing old inactive issues. I just want to better understand first the motivation for this so we can decide whether we want to automate this or whether we just want to clean up issues that are really irrelevant by now.
